projects.yaml

```diff
@@ -82,6 +84,10 @@ settings:
         uvision:
             template:
                 - tools/records/templates/template.uvproj
+        gcc:
+            path:
+                - /opt/gnu-mcu-eclipse/arm-none-eabi-gcc/current/bin/
```

Comment:
This is a user specific, you rather add this to your project or path than in this example